commands/install.cmd

```diff
@@ -74,6 +74,8 @@ if [[ "$OSTYPE" == "linux-gnu" ]] && [[ ! -f "${WARDEN_HOME_DIR}/nodnsconfig" ]]
     echo "==> Configuring resolver for .test domains (requires sudo privileges)"
     if ! sudo grep '^prepend domain-name-servers 127.0.0.1;$' /etc/dhcp/dhclient.conf >/dev/null 2>&1; then
       echo "  + Configuring dhclient to prepend dns with 127.0.0.1 resolver (requires sudo privileges)"
+      ## Ensure that dhclient.conf exists before using it
+      mkdir -p "/etc/dhcp" && touch "/etc/dhcp/dhclient.conf"
```

Comment:
Would it make more sense perhaps to add a condition for file presence to the if conditional which checks for the prepend directive? I.e. don't do anything if the file does not exist. Or is the file not there by default on Arch linux, and once it's there it'll be utilized, one just has to create it first?

Reply:
Yeah by default that file doesn't exist at all in Arch and will be used once the file is created and Network Manager is configured to use dhclient as its DHCP backend.